internal/constants/features.go

```diff
@@ -7,5 +7,6 @@ const (
 	// +deprecated
 	UseOldRemoteSyslogPlugin = "clusterlogging.openshift.io/useoldremotesyslogplugin"
 
-	AnnotationDebugOutput = "logging.openshift.io/debug-output"
+	AnnotationDebugOutput               = "logging.openshift.io/debug-output"
+	AnnotationOCPConsoleMigrationTarget = "logging.openshift.io/ocp-console-migration-target"
```

Comment:
Does it make sense to rename this so it's more clear what it does? For example:

```suggestion
	AnnotationOCPConsoleMigrationTarget = "logging.openshift.io/force-enable-ocp-console-target"
```